.github/workflows/pytx3-ci.yaml

```diff
@@ -38,7 +38,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v2
         with:
-          python-version: "3.x"
+          python-version: "3.8"
```

Comment:
This change is just for CI right? 
Could this at all affect pytx3 usability (or at least signal of compatibility) on <python3.8?

Reply:
Correct, the change is just for CI builds. This is to prevent 3.9 from being used in CI since numpy does not distribute a wheel for 3.9 yet (3.9 having just been released this month). 3.9 can be used with numpy, but the install takes a while on CI.

I'll do a separate PR to add a matrix for testing 3.6 and 3.7 along with 3.8. (3.6 being the lowest version of python that we state we support).